lib/helper/Appium.js

```diff
@@ -836,27 +1022,158 @@ class Appium extends WebdriverIO {
     return this.browser.closeApp();
   }
 
-}
+  /**
+   * {{> ../webapi/appendField }}
+   *
+   */
+  appendField(field, value) {
+    return super.appendField(parseLocator.call(this, field), value);
+  }
 
-function isCSSorXPathLocator(locator) {
-  if (locator[0] === '#' || locator[0] === '.') {
-    if (browserCap) {
-      return true;
-    } else {
-      throw new Error(
-        `Unable to use css locators in apps. Locator strategies for this request: xpath, id, class name`);
-    }
+  /**
+   * {{> ../webapi/checkOption }}
+   *
+   */
+  checkOption(checkbox) {
```

Comment:
Appium and WebDriverIO have a different approach for handling locators. See how `parseLocator` function differs from `wihStrictLocator` in `WebDriverIO`.